simpleast-core/src/main/java/com/agarron/simpleast_core/builder/Parser.java

```diff
@@ -94,6 +92,10 @@ public Parser addRules(final Rule... rules) {
         return root.getChildren();
     }
 
+    private static boolean isTextEmpty(final CharSequence text) {
```

Comment:
I changed it so my unit tests would work. Could have mocked `TextUtils.isEmpty()` but that would have been more trouble than it is worth IMO. Also this is the choice that makes it more pure Java, not that that is likely to ever matter because the chances of me breaking `SimpleRenderer` out into another dep is slim to none.